README.md

```diff
@@ -79,6 +79,10 @@ curl -sSL https://git.io/JfAPE | bash
 
 ### [ ChangeLog (dd/mm/yy)]
 
+### (07/04/24)
+
+· 🐛 FIX: Emulatos > mGBA.
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **issue (typo):** Typo in 'Emulatos', should be 'Emulators'.